specification/personal-demographics.yaml

```diff
@@ -204,12 +204,13 @@ info:
     Errors specific to each API are shown in the Endpoints section, under Response. See our [reference guide](https://digital.nhs.uk/developer/guides-and-documentation/reference-guide#http-status-codes) for more on errors.
     
     ## Open source
-    You might find the following open source resources useful:
+    You might find the following [open source](https://digital.nhs.uk/developer/guides-and-documentation/reference-guide#open-source) resources useful:
 
     | Resource                  | Description                                                          | Links                                                                          |
     |---------------------------|----------------------------------------------------------------------|--------------------------------------------------------------------------------|
     | PDS FHIR API              | Source code for the API proxy, sandbox and specification.            | [GitHub repo](https://github.com/NHSDigital/personal-demographics-service-api) |
     | FHIR libraries and SDKs   | Various open source libraries for integrating with FHIR APIs.        | [FHIR libraries and SDKs](https://digital.nhs.uk/developer/guides-and-documentation/api-technologies-at-nhs-digital#fhir-libraries-and-sdks) |
+    | nhs-number                | Python package containing utilities for NHS numbers including validity checks, normalisation and generation. | [GitHub repo](https://github.com/uk-fci/nhs-number/blob/main/README.md) \| [Python Package index](https://pypi.org/project/nhs-number/) \| [Docs](https://nhs-number.uk-fci.tech/) |
```

Comment:
Would the github repo link be better as: https://github.com/uk-fci/nhs-number/

It seems weird not to link to the repo's homepage. And github renders the readme file on the homepage

Reply:
Agreed - changed